test/extended/openstack/OWNERS

```diff
@@ -0,0 +1,16 @@
+approvers:
+- EmilienM
+- adduarte
+- iamemilio
+- mandre
+- mdbooth
+- pierreprinetti
+- stephenfin
```

Comment:
that are a lot of approvers.

Reply:
This is the team in charge of the component (OpenShift on OpenStack). What is the concern?

Reply:
We loosely follow the definition from upstream: https://github.com/kubernetes/community/blob/master/community-membership.md. Seeing as many reviewers as approvers is unusual.

Reply:
@sttts I don't disagree with you it's important to pay attention keeping the list of approvers up-to-date with active developers in upstream kubernetes, however I think we need to consider what this OWNERS file gives access to. We're talking about unit tests, and an internal team that maintains them.

Reply:
Breaking these tests will break CI for the org and make hundred people unable to merge anything, at least as soon as you move to having an always running e2e job (which I expect you want if your want into origin). So there is quite some responsibility.

Reply:
There is no clear rule about this. I think there is risk and we should take e2e tests serious, especially when they run from origin and break the whole org if something is wrong. My opinion is that 7 (whole team) is wrong, versus a healthy  2 or 3. IMO: start small and add people when they gain experience and have proven to deliver reliable and non-flaky code.

I took the chance and checked the other packages:
```
cmd: reviewers 3, approvers 3
cluster: reviewers 2, approvers 1
router: reviewers 6, approvers 6
cluster: reviewers 2, approvers 1
dr: reviewers 4, approvers 4
operators: reviewers 11, approvers 4
annotate: reviewers 1, approvers 2
image: reviewers 3, approvers 3
image_ecosystem: reviewers 3, approvers 3
images: reviewers 2, approvers 2
networking: reviewers 4, approvers 9
baremetal: reviewers 0, approvers 3
builds: reviewers 3, approvers 3
imageapis: reviewers 2, approvers 2
tbr_health: reviewers 3, approvers 3
etcd: reviewers 4, approvers 4
extended: reviewers 8, approvers 8
prometheus: reviewers 5, approvers 5
rbac: reviewers 4, approvers 0
router: reviewers 6, approvers 6
```

I don't want to block this PR. Delegating to @mfojtik.

Reply:
Fair enough, I get your point and I see that indeed, OWNERS files in origin/tests have a low number of approvers.
It should be fine to keep the whole team as reviewers but we should make the list of approvers shorter. I think we need at least @pierreprinetti and myself in it.

Reply:
With the possible exception of myself, each of the 7 approvers I propose in this PR are talented engineers who bring a demonstrable track record of valid contributions in open source.

I take testing very seriously and I believe that each of these individuals is worth in this repository the same level of trust they have earned over the years in many others across the OpenShift and OpenStack organisations.

Reply:
@pierreprinetti being approver or not has nothing to do with talent or valid contributions in open source. Also nobody want to keep away approver rights from teams. Just looking at the contribution count of the mentioned github handles I see only some with contributions to this repo.

In general, I am not against full ownerships (everybody approver) in team owned repos. But this is a shared code base, with shared maintenance, shared CI pain and potentially impact on a whole org of a hundred developers. Hence, I feel like it is healthy to follow "earned authority" rather than "authority by role".